index.bs

```diff
@@ -461,6 +462,13 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         thousands of samples of audio per second.<br>
         44100 <code>Hz</code> is equivalent to 44.1 <code>kHz</code>.
       </p>
+
+      <p>
+        The <dfn for='AudioConfiguration' dict-member>spatialRendering</dfn> member
+        specifies whether the audio track has the ability to be rendered spatially.
+        The user agent SHOULD only indicate support for an {{AudioConfiguration}} with {{AudioConfiguration/spatialRendering}}
```

Comment:
nit: can you add some line breaks to keep this roughly within an 80 char ruler? Sorry, I know the file has numerous exceptions to this already.

Reply:
Fixed.